benchmarks/annotation_store.ipynb

```diff
@@ -0,0 +1,2103 @@
FAFh8JRo4HycCy0l85Cp9NBVas2gKLehEyhhMLdExACsFigCi6aj+nPS1DKZTClJEPu5gn8dZxCK5ch59cNUPrXhObZNtYzp4TJBJlWh8LCQih8im6RIAqKdj95vDsVvl+tgdvgsTDfzkDml1OgZGgQlYi7p6jC5eXlwbl5JFQhdZHz81oUXjyHvPhD0IZHQhH8FESeEbfnzYC2RRsom4TDIfolZG9YBf0PK1CQlIC8E0eha9UBcidnOL/SD4Zdv+LWzA+g8PSGJesW3Ia9D6PRCEdHRxh2bIElRw/XN0bCkJ8PIQTkcjnurJqPgvNnoKoZjPwzJwEAjp1fBgCkf/A2lD41IHN0giU3BwoPL1h4BhVRiZ6I52nwhoUVr1evXlCpVFi3bl2J47VaLVzVKuRu3wzT1UtQ120AXfsXkGUwwt3JEdk/rYHSvyZkLaJQWFgIJ3Mhcv77M8wZaVA/1QC6Di8i884dODg4QJ11C7n//RmWnGzoIttD0SQcmZmZ8PDwQMGebTCnp8Hplf5I1+thsViKruu4nQ7DwViY01Igd3aD9tlWkIc1gUwmQ+Hh3chP/APCmAuFpy8cO3dDrtbR7lRgqlq8YWHlKeuGhQwNKhf3Cw2g6HTauy/uMxgMsFgs0Gg0UKlUAIouxDObzVAqldDpdJDL5TCZTDAajbBYLAAAtVoNrVZrvbiv+OwolUoFjaboeEhBQQEK/jr9tngajUYDhUIBi8WCgoIC5OXlQS6XQ6vVQqlUQi6Xw2w2Iy8vD4WFhRW1qughMTQqT1mhwd1TFcQ8pFtVl1ChZiddx1cXUu2GBwQE2LweG+qH9+oWbYBmAAV2UwCGEoaZAZR2wqvxr38lTVPy9eVlT/N34kGxZMvfmJro8VNtQ4O3EalY79X1t4YBET05qm1o8DYiRETlj6fcEhGRZAwNIiKSjKFBRESSMTSIiEgyhgYREUnG0CAiIskYGkREJBlDg4iIJGNoEBGRZAwNIiKSjKFBRESSVdt7T5WH9evX2w2rV68emjVrhsLCQmzcuNFufMOGDdGoUSMYzRZsTc20G9/E1RH1nHXQF5rx37TbduOfcXNCHSctMgtM2Hkzy258Cw9nBDlocDO/EHvS79iNb+XpAn+dGteNBThwS283vq23K3w0KvxpyMf/MrPtxkf7uMFDrcTFnDzEZ+XYjX/e1x0uKgXOZRtx8o798yZe8vOATiFHgt6ARL39/Wtf8feESi7DH1m5SMqxv+/sq4FFzw4/djsHl3Nt71mrkMnQI8ATAHD4VjaSjbb3wdXI5ejm7wEA2Jehx40823vqOikV6FrDHQCwO/0O0vNt72/rplKik68bAGBHWhayCk024701KrTzdgUA/HbjNnJMZsju2kb8/PwQFRUFANi8eTPy8mzrr1WrFiIiIgAAP/30E0wm2/nXrl0bzz77LIC/ue0Zjdiyxf7uu02bNkVYWBj0ej1+++03u/HNmzdHaGgoMjMzsWPHDrvxzz33HIKCgnDz5k3ExsbajW/dujUCAgKQkpKC/fv3241v3749fHx88Oeff+Lw4cN24zt16gQPDw9cuHABcXFxduNff/11AMDZs2fxxx9/2I3v1q0bdDodTp8+jYSEBLvxPXr0gEqlwokTJ3Du3Dm78X369AEAHD16FJcuXbIZp1Qq0bNnTwDAoUOHcPXqVZvxWq0WL79c9FCvvXv3IjXV9g7Qzs7OeOGFFwAAsbGxuHnzps14d3d3dO7cGQCwfft23L5t+93g4+OD9u3bAwD+85//IDvb9m+3pG2v+P2Ut2rb0zh27BgWLVpU1WUQEVUrfAhTBanuz9OgInyeRuXhQ5gqT1kPYaq2PQ0iIip/DA0iIpKMoUFERJIxNIiISDKGBhERScbQICIiyRgaREQkGUODiIgkY2gQEZFkDA0iIpKMoUFERJIxNIiISDKGBhERScbQICIiyaptaPB5GkRE5a/aPrkvPDwc4eHhVV0GEVG1Um17GkREVP4YGkREJBlDg4iIJGNoEBGRZAwNIiKSjKFBRESSMTSIiEgyhgYREUnG0CAiIskYGkREJBlDg4iIJGNoEBGRZAwNIiKSjKFBRESSMTSIiEgyhgYREUnG0CAiIskYGkREJBlDg4iIJKu2oXHs2DEsWrSoqssgIqpWlFVdQEUJDw9HeHh4VZdBRFStVNueBhERlT+GBhERScbQICIiyRgaREQkGUODiIgkY2gQEZFkDA0iIpKMoUFERJIxNIiISDKGBhERScbQICIiyRgaREQkGUODiIgkY2gQEZFkDA0iIpKMoUFERJIxNIiISLIyn9yn1+uxd+9exMfH488//4TBYICDgwOCgoLQrFkztGvXDi4uLpVVKxERVbFSQ2Pt2rXYt28fnn76aXTo0AEBAQHQ6XQwGo1ISUlBYmIiJk6ciNatW6N///6VWTMREVWRUkPD3d0dc+fOhUqlshsXEhKC1q1bo6CgALt27arQAomI6NFRamh07dr1vhOr1Wp06dKlXAsiIqJHV5nHNIqdPn0aPj4+8PHxwe3bt/Hdd99BLpejX79+cHNzq+ASiYjoUSHp7Klly5ZBLi9qunr1apjNZshkMixatKhCiyMiokeLpJ5GZmYmvLy8YDab8ccff+Drr7+GUqnEsGHDKro+IiJ6hEgKDZ1Oh6ysLCQnJyMwMBBarRYmkwkmk6mi6yMiokeIpNDo0qULJk2aBJPJhEGDBgEAzp49i4CAgIqsjYiIHjGSQuOVV15BixYtIJfLUaNGDQCAh4cHhg8fXqHFERHRo0VSaACAv79/ma+JiKj6K/XsqUmTJuHQoUOlHrcwmUw4ePAgJk+eXGHF/R3Hjh3j2V1EROVMJoQQJY24du0a1q9fj8TERISEhMDf3x9arRZ5eXlITU3FpUuX0KhRI7z66qsIDAys7LofyPXr1yt9meYh3Sp9mVT5FEu2VHUJTwwvLy9kZGRUdRlPhLL2JJUaGsWysrJw8uRJXL16Fbm5u
```

Comment:
1) Is there a replacement for wget that is part of standard bash on the usual platforms?
2) Could the error message be more enlightening? Something like 
On Macos: EITHER brew install wget OR sudo port install wget
Perhaps that would already be obvious to users.

Reply:
We could possibly swap to curl which is more commonly pre-installed or use requests (which tiatoolbox installs as a dependency) and do it in python.